compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -10438,11 +10436,11 @@ OMR::Z::TreeEvaluator::arraycmpHelper(TR::Node *node,
    {
    TR::Compilation *comp = cg->comp();
 
-   if (TR::isJ9() && !comp->getOption(TR_DisableSIMDArrayCompare) && cg->getSupportsVectorRegisters() && !isWideChar)
```

Comment:
Checking out the code for `isWideChar`, I see that we double the length for the operand, which may suggests that the code was added to handle comparison of Strings that are UTF-16.   @joransiu  Do you remember why we had this `isWideChar` code ? Checking the current code, it is set to false always and I do not see other codegen doing anything special for this. Want to make sure it is safe to remove it. 

Tagging @jdmpapin / @dchopra001  here as well , if they have any more context behind this.